ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_adapter_config.schema.yml

```diff
@@ -455,6 +455,9 @@ $defs:
             description: |
               Port-Channel administrative state.
               Setting to false will set port to 'shutdown' in intended configuration.
+          ptp_mpass:
```

Comment:
We should have a description here. Check the TOI https://www.arista.com/en/support/toi/eos-4-26-2f/14803-ptp-mlag and write a good useful description. It is important we include the detail `MPASS should be configured only on port-channels connecting to non-Arista devices.`

Reply:
Done!

Reply:
Nope not done :)